PinpointKit/PinpointKit/Sources/FeedbackViewController.swift

```diff
@@ -153,7 +153,12 @@ public final class FeedbackViewController: UITableViewController {
         }
         
         cancelBarButtonItem.setTitleTextAttributes([NSFontAttributeName: appearance.feedbackCancelButtonFont], forState: .Normal)
-        navigationItem.leftBarButtonItem = cancelBarButtonItem
+        
+        if viewController.presentingViewController != nil {
```

Comment:
Here we check `viewController.presentingViewController`, but in `cancelButtonTapped()` we just check `presentingViewController`. We should keep it consistent.